crates/cxx-qt-gen/src/parser/mod.rs

```diff
@@ -100,6 +128,16 @@ impl Parser {
             ));
         }
 
+        // Maybe not the best but removes use of take_path
+        let filtered: Vec<Attribute> = module
+            .attrs
+            .iter()
+            .filter(|attr| !path_compare_str(attr.meta.path(), &["cxx_qt", "bridge"]))
+            .cloned()
+            .collect();
+
+        parse_attributes(&filtered, &["doc"])?;
```

Comment:
ah yes would be nice to make `parse_attributes` support longer paths like `cxx_qt::bridge`. Could you make `parse_attributes` check if the string contains a `::` if it does then split it and use path_compare_str on the list if it doesn't then just do `[string]` like it does currently?

Reply:
Done, added a path split helper function which might be useful elsewhere too